tests/Integration/Database/EloquentWhereHasMorphTest.php

```diff
@@ -53,7 +53,7 @@ public function testWhereHasMorph()
     {
         $comments = Comment::whereHasMorph('commentable', [Post::class, Video::class], function (Builder $query) {
             $query->where('title', 'foo');
-        })->get();
+        })->orderBy('id')->get();
```

Comment:
SQLite, MySQL 5.7 & MySQL 8.0 have different behavior for ordering the result set in these tests. Since the focus of the tests isn't on the order of the results, I've added `orderBy` clauses to all of these queries.